### PR TITLE
Update contributing instructions to include pandoc 

### PR DESCRIPTION
**Pull request by gsheni**
- Update Dockerfile instructions
- Add pandoc to contributing.md

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -61,7 +63,7 @@ Before starting major work, you should touch base with the maintainers of Featur
 
      **Ubuntu**:
      ```console
-     sudo apt install openjdk-11-jre openjdk-11-jdk scala graphviz -y
+     sudo apt install openjdk-11-jre openjdk-11-jdk scala pandoc -y
```
Comment (thehomebrewnerd) [inline contributing.md:66]: Are the graphviz requirements different for Ubuntu than they are for macOS? We removed graphviz here, but not for the macOS instructions above.

Comment (gsheni) [reply contributing.md]: This was a mistake. Fixed

**Comment by gsheni** [comment]
@thehomebrewnerd We **always** needed to install pandoc to build the docs. This just adds it to the contributing doc

**Comment by thehomebrewnerd** [comment]
> @thehomebrewnerd We **always** needed to install pandoc to build the docs. This just adds it to the contributing doc

Hmm...ok. I never recall installing it previously, but maybe I have just forgotten. I definitely can't build now without installing though.

File: contributing.md [hunk 2]
```diff
@@ -61,7 +63,7 @@ Before starting major work, you should touch base with the maintainers of Featur
 
      **Ubuntu**:
      ```console
-     sudo apt install openjdk-11-jre openjdk-11-jdk scala graphviz -y
+     sudo apt install openjdk-11-jre openjdk-11-jdk scala pandoc pandoc -y
```
Comment (thehomebrewnerd) [inline contributing.md:66]: pandoc is listed twice - one should be graphviz

Comment (gsheni) [reply contributing.md]: Fixed

**Review by thehomebrewnerd — APPROVED** [review]
I'm not well-versed enough in Docker to know if the updates for the dockerfile all good, but they seem ok to me.

File: contributing.md [hunk 3]
```diff
@@ -42,6 +42,8 @@ Before starting major work, you should touch base with the maintainers of Featur
 
 * You will need to install Spark, Scala, GraphViz, and Pandoc to run all unit tests & build docs:
 
+  > If you do not install Spark/Scala, you can still run the unit tests (the Spark tests will be skipped).
+
```
Comment (rwedge) [inline contributing.md:46]: Should we add another note stating that Pandoc is only needed for building the docs?

Comment (gsheni) [reply contributing.md:47]: Done

**Review by rwedge — APPROVED** [review]
Docker image worked for me!

**Review by rwedge — APPROVED** [review]

**Pull request merged**